_sass/3-layout/_global.scss

```diff
@@ -0,0 +1,103 @@
+@import "./0-base/reset";
+@import "./1-modules/mixins";
+@import "./1-modules/colors";
+@import "./1-modules/selectBox";
+@import "./1-modules/stickyAside";
+
+@import "./list";
+@import "./email";
+
+body {
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  -moz-font-feature-settings: "kern", "liga", "clig", "calt";
+  -ms-font-feature-settings: "kern", "liga", "clig", "calt";
+  -webkit-font-feature-settings: "kern", "liga", "clig", "calt";
+  font-feature-settings: "kern", "liga", "clig", "calt";
+  background-color: var(--bg);
+  font-family: 'Source Code Pro', 'Helvetica', 'Arial', sans-serif;
+  font-size: 16px;
```

Comment:
Possibly out of scope of this change, but perhaps we want to use relative units on font sizes so they respect the users font size preferences?
<img width="696" alt="Screen Shot 2020-06-12 at 1 08 01 PM" src="https://user-images.githubusercontent.com/10820686/84546393-c143ff00-acad-11ea-9887-3b424e0b67e0.png">

Reply:
super good idea and i think it fits the scope of this PR since it already touches everything else with the style system~

Reply:
DONE! do you have a preferred scale? Major third, etc?